Library/Formula/weboob.rb

```diff
@@ -0,0 +1,52 @@
+class Weboob < Formula
+  desc "Web Outside of Browsers"
+  homepage "http://weboob.org/"
+  url "https://symlink.me/attachments/download/289/weboob-1.0.tar.gz"
+  sha256 "2500823b6de62161d4da11382181f5def0d91823b23cebd9a470479714844068"
+  head "git://git.symlink.me/pub/weboob/stable.git"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on "libyaml"
+  depends_on "gnupg"
```

Comment:
Might be able to use the special `depends_on :gpg` dependency then. It defaults to `gnupg` rather than `gnupg2` but I may look at switching that around after the holidays.